tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_extcode.py

```diff
@@ -0,0 +1,92 @@
+"""
+test execution semantics changes
+"""
+from ethereum.crypto.hash import keccak256
+
+from ethereum_test_tools import (
+    Account,
+    Address,
+    Environment,
+    StateTestFiller,
+    Storage,
+    TestAddress,
+    Transaction,
+)
+from ethereum_test_tools.eof.v1 import Container, Section
+from ethereum_test_tools.eof.v1.constants import NON_RETURNING_SECTION
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+REFERENCE_SPEC_GIT_PATH = "EIPS/eip-4750.md"
+REFERENCE_SPEC_VERSION = "90f716078d0b08ce508a1e57803f885cc2f2e15e"
+
+
+def test_legacy_calls_eof_sstore(
+    state_test: StateTestFiller,
+):
+    """Test EXTCODE* opcodes calling EOF and legacy contracts"""
+    env = Environment()
+    address_eof_contract = Address(0x1000000)
+    address_legacy_contract = Address(0x1000001)
+    address_test_contract = Address(0x1000002)
+    storage_test = Storage()
+
+    legacy_code = Op.PUSH1(2) + Op.JUMPDEST + Op.STOP
+    pre = {
+        TestAddress: Account(
+            balance=1000000000000000000000,
+            nonce=1,
+        ),
+        address_eof_contract: Account(
+            code=Container(
+                sections=[
+                    Section.Code(code=Op.RJUMP[0] + Op.STOP, code_output=NON_RETURNING_SECTION)
```

Comment:
```suggestion
                    Section.Code(code=Op.RJUMP[0] + Op.STOP, code_outputs=NON_RETURNING_SECTION)
```
It took me 20 minutes to find out this typo was the reason why we were getting `INVALID_FIRST_SECTION_TYPE` in this test.

I think we might need to add the proper keyword arguments (with defaults) to `Section.Code`.